mineinabyss-components/src/main/kotlin/com/mineinabyss/components/gondolas/Gondola.kt

```diff
@@ -10,7 +10,14 @@ import org.bukkit.Location
 @SerialName("mineinabyss:gondola")
 class Gondola(
     @Serializable(with = LocationSerializer::class)
-    val location: Location,
-    val name: String,
-    val displayItem: SerializableItemStack,
-)
+    val upperLoc: Location,
+    @Serializable(with = LocationSerializer::class)
+    val lowerLoc: Location,
+    val name: String, // the name of the gondola
+    val unlockPrice: Int, // the price needed to unlock access to the line
+    val displayItem: SerializableItemStack, // the item to display in the GUI
+    val warpZoneRange: Double, // the range in which the player needs to stay in order to be teleported
+    val noAccessMessage: String, // the message to display when the player tries to use the gondola without perms
```

Comment:
`@SeirlaName("noAccessMessage") private val _noAccessMessage: String`
then inside {} add `@Transient val noAccessMessage: Component = _noAccessMessage.miniMsg()`
as we wanna use components for most things internally